corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -384,13 +389,16 @@ hqDefine("app_manager/js/details/column", function () {
                         fireChange();
                     });
                     self.date_extra.value = format.val();
-                } else if (this.val() === "enum" || this.val() === "enum-image" || this.val() === 'conditional-enum') {
+                } else if (this.val() === "enum" || this.val() === "enum-image"
+                           || this.val() === 'conditional-enum' || this.val() === 'translatable-enum') {
                     self.enum_extra.values_are_icons(this.val() === 'enum-image');
                     self.enum_extra.keys_are_conditions(this.val() === 'conditional-enum');
+                    self.enum_extra.values_are_translatable(this.val() === 'translatable-enum');
                     self.format.ui.parent().append(self.enum_extra.ui);
                 } else if (this.val() === "clickable-icon") {
                     self.enum_extra.values_are_icons(true);
                     self.enum_extra.keys_are_conditions(true);
+                    self.enum_extra.values_are_translatable(true);
```

Comment:
what needs to be translated for clickable icons? The alt text?

Reply:
I need to double check since it was a month ago but I think I forgot to take that line out